scripts/qwen_tts_api.py

```
@@ -0,0 +1,308 @@
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+)
+logger = logging.getLogger(__name__)
+
+# Environment variables
+MODEL_NAME = os.getenv("QWEN_TTS_MODEL", "Qwen/Qwen3-TTS-12Hz-1.7B-CustomVoice")
+DEVICE = os.getenv("QWEN_TTS_DEVICE", "cuda:0" if torch.cuda.is_available() else "cpu")
+DTYPE = os.getenv("QWEN_TTS_DTYPE", "bfloat16")
+PORT = int(os.getenv("QWEN_TTS_PORT", "8890"))
+HOST = os.getenv("QWEN_TTS_HOST", "0.0.0.0")
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)